lib/client/rpc/proposals/finalizeProposal.ts

```diff
@@ -129,6 +131,11 @@ export class FinalizeProposalClient implements FinalizeProposal {
       const proposalAccount = await this.autocratClient.getProposal(
         proposal.publicKey
       );
+      const crankFailIx = await InstructionUtils.getInstructions(this.autocratClient.ammClient.crankThatTwapIx(proposal.failMarket))
+      const crankPassIx = await InstructionUtils.getInstructions(this.autocratClient.ammClient.crankThatTwapIx(proposal.passMarket))
+
+      const crankTx = new Transaction().add(...crankFailIx, ...crankPassIx)
```

Comment:
Sorry what I meant was to include these IXes in the finalize proposal instruction so it's one and the same. Not two transactions. Each crank is only 5k CUs and then the finalize for memo program is only 50k compute.

https://explorer.solana.com/tx/Dhnj1Rqi4qYK6p4XFmhydmFiSs8fxjiURyZSwFagntyR5c5tMLGqJvooK9rWxeasaEPKuC6V7jB9jf8wFg8d5ec?cluster=triton

So this should be more than enough to include in the finalize txn. So its fail or succeed.

Reply:
I included it in another tx because of errors parsing, since error codes overlap : amm is between 6000 and 6016 and autocrat between 6000 and 6010

Reply:
Will look at workarounds, for for now I put it back in just a tx